### PR TITLE
Patch to fix Network_config script

### PR DESCRIPTION
**Pull request by hathyaga**
Fixes mtu match scenario for the network interface

Signed-off-by: Harsha Thyagaraja <harshkid@linux.vnet.ibm.com>

### CODE REVIEW

**Comment by hathyaga** [comment]
[network_config_PASS.txt](https://github.com/avocado-framework-tests/avocado-misc-tests/files/918616/network_config_PASS.txt)

**Comment by PraveenPenguin** [comment]
@hathyaga can you please more elaborate in commit message why this is required ?

File: io/net/Network_config.py [hunk 1]
```diff
@@ -110,7 +110,8 @@ def test_networkconfig(self):
                     self.fail("mismatch in mtu")
             if 'MTU' in val:
                 cmd, mtu_val = val.split('MTU:')
-                mtu_val, cmd = mtu_val.split(' ')
+                mtu_val, cmd = mtu_val.split(' M')
```
Comment (narasimhan-v) [inline io/net/Network_config.py]: Can you explain with example why this was not working before and why it would work now please

Comment (hathyaga) [inline io/net/Network_config.py]: The script would error out at this line: mtu_val, cmd = mtu_val.split(' ') 
with the error "ERROR| ValueError: too many values to unpack" as it was retriving the MTU vaue as "1500  Metric:1"

In Rhel and Ubuntu, the 'mtu' value is displayed as "mtu 1500" 
In Sles, it is seen as "MTU:1500  Metric:1" 

So when Iterating in the list and finding the mtu value, With the patch, the value before ' M' where 'M is Metric, is put in mtu_val which is the value we need.
cmd, mtu_val = val.split('MTU:')
mtu_val, cmd = mtu_val.split(' M')
mtu_val = mtu_val.strip()

Comment (hathyaga) [inline io/net/Network_config.py]: Does that help?

Comment (narasimhan-v) [inline io/net/Network_config.py]: This can be (in fact, should be) handled with regex

Comment (narasimhan-v) [inline io/net/Network_config.py]: I do not find any difference between SLES and Ubuntu.

### SLES: ###
```
linux-k12h:~ # ifconfig eth0 | grep MTU
          UP BROADCAST RUNNING MULTICAST  MTU:1500  Metric:1

linux-k12h:~ # ip link show eth0
2: eth0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc pfifo_fast state UNKNOWN mode DEFAULT group default qlen 1000
    link/ether ea:e3:86:8d:2f:02 brd ff:ff:ff:ff:ff:ff
```

### UBUNTU: ###
```
# ifconfig wc0 | grep MTU
          UP POINTOPOINT RUNNING NOARP MULTICAST  MTU:1322  Metric:1

# ip link show wc0
9: wc0: <POINTOPOINT,MULTICAST,NOARP,UP,LOWER_UP> mtu 1322 qdisc pfifo_fast state UNKNOWN mode DEFAULT group default qlen 500
    link/none 
```

**Comment by hathyaga** [comment]
Closing this as a new PR https://github.com/avocado-framework-tests/avocado-misc-tests/pull/443 is submitted and ,merged

**Pull request closed without merging**